src/SplashScreen/SplashNav/SplashNavStyles.ts

```diff
@@ -39,7 +39,7 @@ export const NavLink = styled('a')`
 export const Logo = styled('img')`
   float: left;
   padding: 20px 0;
-  width: '65%';
+  width: 65%;
   height: 'auto';
```

Comment:
This should also not be a string like line 42.
```suggestion
  height: auto;
```